docs/external-resources.md

```diff
@@ -155,8 +155,9 @@ _Note: Each of [our SDK docs](/application-development) have a Hello World and R
 
 ## Community SDKs
```

Comment:
@rachfop Let's just move this list to this section https://docs.temporal.io/temporal#third-party-sdks and remove this from external resources.
Long game is to break up the content on this page, redistribute it, and remove this page.